File: BankingAppObjectiveC/BankingApp/WebViewController.m
//
//  WebViewController.m
//  BankingApp
//
//  Created by Aasveen Kaur on 2/23/17.
//  Copyright © 2017 Jason Kim. All rights reserved.
//

#import "WebViewController.h"

@interface WebViewController ()
@property (weak, nonatomic) IBOutlet UIWebView *myWebView;
@property (weak, nonatomic) IBOutlet UIActivityIndicatorView *myActivityIndicator;

@end

@implementation WebViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    _myWebView.delegate = self;
   NSString *urlString = @"https://msdn.microsoft.com/en-us/skype/appsdk/gettingstarted#next-steps";
    NSURL *url = [NSURL URLWithString:urlString];
    NSURLRequest *urlRequest = [NSURLRequest requestWithURL:url];
    [_myWebView loadRequest:urlRequest];

}

-(void)webViewDidFinishLoad:(UIWebView *)webView{
    if (_myWebView.isLoading)
        return;
        
        [_myActivityIndicator stopAnimating];
    
}

@end

File: BankingAppSwift/BankingAppSwift/WebViewController.swift
//
//  WebViewController.swift
//  BankingAppSwift
//
//  Created by Aasveen Kaur on 2/23/17.
//  Copyright © 2017 Aasveen Kaur. All rights reserved.
//

import UIKit

class WebViewController: UIViewController,UIWebViewDelegate {

    @IBOutlet weak var myActivityIndicator: UIActivityIndicatorView!
    @IBOutlet weak var myWebView: UIWebView!
    override func viewDidLoad() {
        super.viewDidLoad()
        myWebView.delegate = self
        let fullURL = "https://msdn.microsoft.com/en-us/skype/appsdk/gettingstarted#next-steps"
        let url = NSURL(string:fullURL)
        let request = NSURLRequest(URL: url!)
        myWebView.loadRequest(request)
        
        // Do any additional setup after loading the view.
    }
    
    func webViewDidFinishLoad(webView: UIWebView) {
        if (myWebView.loading){
        return
        }
       myActivityIndicator.stopAnimating()
       
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}
